src/API.jl

```diff
@@ -718,12 +726,12 @@ function instantiate(ctx::Context; manifest::Union{Bool, Nothing}=nothing,
         Types.write_project(Dict("deps" => deps), ctx.env.project_file)
         return instantiate(Context(); manifest=manifest, update_registry=update_registry, verbose=verbose, kwargs...)
     end
-    if (!isfile(ctx.env.manifest_file) && manifest == nothing) || manifest == false
+    if (!isfile(ctx.env.manifest_file) && manifest === nothing) || manifest == false
         up(ctx; update_registry=update_registry)
         return
     end
     if !isfile(ctx.env.manifest_file) && manifest == true
-        pkgerror("manifest at $(ctx.env.manifest_file) does not exist")
+        pkgerror("Manifest at `$(ctx.env.manifest_file)` does not exist.")
```

Comment:
Not a big deal but I typically like to start with non-capital letters and base does that a lot too, e.g.:

```
julia> [][1]
ERROR: BoundsError: attempt to access ..
```

Reply:
Sure, I mostly just care about the consistency. I'm planning a separate PR to make formatting for the error messages we have consistent, I can change it then